modules/smart-agent_kubernetes-workloads-count/conf/readme.yaml

```
@@ -0,0 +1,22 @@
+documentations:
```

what we are interested here is the monitor used to collect metrics

```suggestion
documentations:
  - name: Smart Agent monitor
    url: 'https://docs.signalfx.com/en/latest/integrations/agent/monitors/kubernetes-cluster.html'
```